back/engines/commercial/idea_custom_fields/app/controllers/idea_custom_fields/web_api/v1/idea_custom_fields_controller.rb

```diff
@@ -160,7 +160,7 @@ def update_fields!(page_temp_ids_to_ids_mapping, option_temp_ids_to_ids_mapping)
           end
           update_statements! field, statements_params, errors, index if statements_params
           relate_map_config_to_field(field, field_params, errors, index)
-          field.set_list_position(index)
+          field.move_to_bottom
```

Comment:
Does this work fine where new fields are added in the middle of the form? I have it in my mind that we changed update_all a while back so that it was not making updates to every field, every time. ie the `next` statements on line 152, 155

Reply:
@jamesspeake I wasn't aware of this change. Will do more testing around these cases (new field in the middle and the other field) and will likely use a different method to insert at that position. But it would have to be a different method because `set_list_position` will temporarily have two fields with the same ordering.